### ISSUE TITLE
[BUG] using alias in response schema

### ISSUE THREAD

**Issue by sebdiem** [tiangolo/fastapi#136]
**Describe the bug**
When using alias in response schema, the documentation uses the alias in the response but the returned response does not use the alias.

**To Reproduce**

```python
class Test(BaseModel):
    test: int = Schema(..., alias=test2)

@router.get("/test", response_model=Test)
def test():
    return {"test": 1}

# in test.py
>> response = client.get("/test", …)
>> response.json()
{"test2": 1}
```

**Expected behavior**
I would expect the doc and the code to be consistent.

**Screenshots**

**Environment:**
 - OS: macOS 
 - FastAPI Version 0.10.1:

```Python
import fastapi
print(fastapi.__version__)
```

- Python version, get it with: 3.7.2

```bash
python --version
```

**Additional context**
N/A

**Comment by sebdiem** [comment]
I can use https://github.com/tiangolo/fastapi/issues/31 as a backup solution but still it seems like a bug to me.

**Comment by tiangolo** [comment]
I agree with you.

I just implemented it in PR #168, including tests.

It is now available in version `0.17.0` (just released).

**Comment by sebdiem** [comment]
Thanks !
